README.md

```diff
@@ -26,26 +35,42 @@ If you are using Maven to manage your project's dependencies, you can run `mvn i
-the `target` directory created when you ran `mvn package`.
+_NOTE:_ Depending on what version of the Java compiler you have, you may need to update the version. At this time, the {version} most compatible is `1.6`:
+
+```xml
+<properties>
+    <maven.compiler.source>{version}</maven.compiler.source>
+    <maven.compiler.target>{version}</maven.compiler.target>
+</properties>
+```
+
+3. Run `mvn install` to install packages and their dependencies locally.
 
-Note that we ran `mvn package` without running the integration tests, since these require access to a Conjur instance. You can run the
-integration tests with `mvn package` once you finished with the configuration.
+_NOTE:_ that we ran mvn package without running the integration tests, since these require access to a Conjur instance. You can run the integration tests 
+with mvn package once you finished with the configuration.
 
-### Configuration
+If generating a JAR is preferred, you can build the library locally and add the dependency to the project manually by following the setup steps below:
 
-The simplest way to configure the Conjur API is by using environment variables, which is often a bit more convenient.
-Environment variables are mapped to configuration variables by prepending `CONJUR_` to the all-caps name of the
+1. Clone the Conjur Java API repo locally: `git clone {repo}`
+2. `cd conjur-api-java`
+3. Run `mvn package -DskipTests` to generate a JAR file. These files should be in the `target` directory of the repo
+4. Follow the steps outlined [here](https://docs.oracle.com/cd/E13224_01/wlw/docs100/guide/ideuserguide/conJARLibraryDependencies.html) for information on how add JAR files into the new Java app's project files 
+
+## Configuration
+Once the setup steps have been successfully run, we will now define the environment variables needed to make the connection between the new Java application and Conjur. The best way to do so is by setting environment variables. 
+In Conjur (both OSS and Enterprise), environment variables are mapped to configuration variables by prepending `CONJUR_` to the all-caps name of the
 configuration variable. For example, `appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.  
 
-The following environment variables are mandatory for running the API: CONJUR_ACCOUNT, CONJUR_AUTHN_LOGIN, CONJUR_AUTHN_API_KEY & CONJUR_APPLIANCE_URL.
+_NOTE:_ For ways to set credentials (for CONJUR_AUTHN_LOGIN and CONJUR_AUTHN_API_KEY environment variables ONLY) in the Java app instead of environment variables, see the [Authorization Examples](#authorization-examples) section below.
+
+The following environment variables need to be included in the apps runtime environment in order use the Conjur API: CONJUR_ACCOUNT, CONJUR_AUTHN_LOGIN, CONJUR_AUTHN_API_KEY & CONJUR_APPLIANCE_URL.
 
 CONJUR_ACCOUNT - account specified during Conjur setup
-CONJUR_APPLIANCE_URL - Conjur HTTPS endpoint
+CONJUR_APPLIANCE_URL - Conjur HTTPS endpoint (OSS/Appliance)
```

Comment:
Use the name of our EE product consistently - in some places it's DAP, in some it's appliance, and in some it's enterprise Conjur